src/main/webapp/app/pages/AccountPage.tsx

```diff
@@ -52,11 +57,18 @@ export const InfoRow: React.FunctionComponent<{
 
 @inject('authenticationStore')
 @observer
-export class AccountPage extends React.Component<IRegisterProps> {
+export class AccountPage extends React.Component<
+  IRegisterProps,
+  IApiAccessRequestState
+> {
   @observable copiedIdToken = false;
+  @observable showConfirmModal = false;
 
   constructor(props: Readonly<IRegisterProps>) {
     super(props);
+    this.state = {
+      apiAccessJustification: '',
```

Comment:
1. `apiAccessJustification` does not need to be a state. 
2. We use Mobx observable for state management in Class. See `showConfirmModal`.